network-api/networkapi/wagtailpages/templates/buyersguide/product_page.html

```diff
@@ -235,9 +235,9 @@ <h3 class="tw-mb-4 tw-font-zilla tw-text-2xl tw-leading-7">{% trans "What data d
                 </div>
 
                 <div class="tw-w-full tw-flex-shrink-0" data-product-label="1">
-                  {% localizedroutablepageurl home_page 'min-security-view' as min_sec_url %}
+                  {% localizedroutablepageurl home_page 'methodology-view' as min_sec_url %}
                   {% blocktrans with url=min_sec_url asvar minimum_security_standards trimmed %}
-                    Does this product meet our <a id="mss-link" class="tw-ml-1" href="{{ url }}">Minimum Security Standards</a>?
+                    Does this product meet our <a id="mss-link" class="tw-ml-1" href="{{ url }}#minimum-security-standards">Minimum Security Standards</a>?
```

Comment:
With the suggestion above, this can get removed

```suggestion
                    Does this product meet our <a id="mss-link" class="tw-ml-1" href="{{ url }}">Minimum Security Standards</a>?
```